alternate-installation-options/index.md

```diff
@@ -31,14 +34,53 @@ $ kubectl apply -k 'github.com/cryostatio/cryostat-operator//config/default?ref=
     $ operator-sdk run bundle quay.io/cryostat/cryostat-operator-bundle:{{ site.data.versions.cryostat.version }}
     ```
 
+### [Using ClusterCryostats](#using-clustercryostats)
+In [Deploying Cryostat](/get-started#deploying-cryostat), you created a single-namespace `Cryostat` Custom Resource
+(`CR`) instance.
+
+Single-namespace `Cryostat` `CR`s instruct the Operator to deploy restricted Cryostat instances which are only able
+to see target applications deployed in the same namespace as the Cryostat instance, which is the same Namespace that
+the `CR` is created within.
+
+If you chose to install the Operator in **All Namespaces** mode, you may also be interested in
+creating `CluterCryostat` `CR`s. In this configuration, the Operator is able to see `Cryostat` and `ClusterCryostat`
+`CR`s in any project (`Namespace`) and create Cryostat deployments corresponding to either `CR` kind.
```

Comment:
Not sure but the users might be confused that `ClusterCryostat` is a namespaced resource here.

Reply:
Maybe. Would it help if I add an explanation for what that is the case?

Reply:
Yeh sure a short explanation should work :))